docker-compose.yml

```diff
@@ -91,3 +91,7 @@ services:
       - type: bind
         source: ./
         target: /app
+      - node-modules:/app/node_modules
+
+volumes:
+  node-modules:
```

Comment:
Seems wrong to me that the file ends here, this works?

Reply:
Yes. I copied that from an official doc. I think it means "create a volume called `node-modules` with all default settings"?